File: LV1/1.7/ostgotska/ostgotska.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
)

func main() {
	var s, ae int
	scanner := bufio.NewScanner(os.Stdin)

	scanner.Scan()
	tmp := strings.Split(scanner.Text(), " ")

	for _, w := range tmp {
		s++
		if strings.Contains(w, "ae") {
			ae++
		}
	}

	if 10*ae >= 4*s {
		fmt.Println("dae ae ju traeligt va")
	} else {
		fmt.Println("haer talar vi rikssvenska")
	}
}

File: LV1/1.7/ostgotska/ostgotska.rs
use std::io::{self, BufRead};

fn main() {
    let stdin = io::stdin();

    let mut buff = String::new();
    stdin.lock().read_line(&mut buff).unwrap();

    let words = buff.split_whitespace().collect::<Vec<&str>>();
    let ae = words.iter().filter(|w| w.contains("ae")).count();

    if ae * 10 >= &words.len() * 4 {
        println!("dae ae ju traeligt va");
    } else {
        println!("haer talar vi rikssvenska");
    }
}
